game_options.cpp

```diff
@@ -147,18 +150,23 @@ void test_game_options()
   }
   #endif // FIX_ISSUE_303
 
-  // #define FIX_ISSUE_353
+  #define FIX_ISSUE_353
   #ifdef FIX_ISSUE_353
   {
     // Random game options should not draw keys already used by another player
-    const int rng_seed = 271; // by default this will assign A to both players
-    const game_options rgo = get_random_game_options(rng_seed);
-    key_action_map map_1 = rgo.get_kam_1();
-    key_action_map map_2 = rgo.get_kam_2();
-    bool a = map_1.has_key(sf::Keyboard::Key::A);
-    bool b = map_2.has_key(sf::Keyboard::Key::A);
-    // Key A is in KAM1 or KAM2 but not both
-    assert(a != b);
+    const int rng_seed = 271;
+    for (int seed = 0; seed < 1000; seed++)

```

Comment:
I enjoy the thoroughness of 1000 iterations! Still, this costs the project in the long run, as everytime the test suit is run, there are 999 superfluous iterations. Also, the feeling you got when picking 1000, i.e. 'Why 1000? Why no 100? Or 10? Or 10000?' is an indication that this was an arbitrary decision. 1 time is good enough here :-)

Reply:
Done!